### ISSUE TITLE
new laxar-info task to inspect project artifacts

### ISSUE THREAD

**Issue by x1B** [LaxarJS/grunt-laxar#72]
We should add a new task `laxar-info` that helps to list and inspect LaxarJS application artifacts through a CLI.

- [x] `--usage`
- [x] `--flow`
- [x] `--page`
- [x] `--layout`
- [x] `--widget`
- [x] `--control`

**Comment by x1B** [comment]
The new `laxar-info` task will usually be aliased to `info` so that it can be invoked like this:

```sh
# print general project information
grunt info

# print usage information
grunt info --usage

# print information on the flow-target _main_
grunt info --flow main

# print information on the widget _cat/my-widget_
grunt info --widget cat/my-widget
```

**Comment by x1B** [comment]
Implemented on `master` (`v1.1.0`).

### Upgrade Information

The new grunt-task `laxar-info` has been added to *grunt-laxar*.
Details can be found in the [`laxar-info documentation`](https://github.com/LaxarJS/grunt-laxar/blob/master/docs/tasks/laxar-info.md).
